core/rpc/api/src/author/mod.rs

```diff
@@ -83,4 +83,24 @@ pub trait AuthorApi<Hash, BlockHash> {
 		metadata: Option<Self::Metadata>,
 		id: SubscriptionId
 	) -> Result<bool>;
+
+	/// Watch multiple extrinsics (own or from network)
+	#[pubsub(
+		subscription = "author_extrinsicTrack",
+		subscribe,
+		name = "author_trackExtrinsic"
+	)]
+	fn track_extrinsic(&self,
+    	metadata: Self::Metadata,
+		subscriber: Subscriber<Status<Hash, BlockHash>>,
+	    hash: Hash,
+	);
+
+	/// Unsubscribe from multiple extrinsics watching.
+	#[pubsub(
+		subscription = "author_extrinsicTrack",
```

Comment:
Yeah, it really is a pitty that we don't support two subscriptions with the same name. Opened an issue for this: https://github.com/paritytech/jsonrpc/issues/511